tests/integration/deploy/test_deploy_command.py

```diff
@@ -1197,29 +1197,3 @@ def test_deploy_logs_warning_with_cdk_project(self):
         deploy_process_execute = run_command(deploy_command_list)
         self.assertIn(warning_message, deploy_process_execute.stdout)
         self.assertEqual(deploy_process_execute.process.returncode, 0)
-
-    def _method_to_stack_name(self, method_name):
```

Comment:
Why we don't need this anymore?

Reply:
Same as above, why do we delete these methods?

Reply:
It is already in package's base class, this is a duplicate and missed imports caused some failures

Reply:
I refactored these methods to be shared together by sync and deploy through package base